basic/openssl.cnf

```diff
@@ -59,12 +59,14 @@ basicConstraints = critical,CA:true
 basicConstraints = CA:false
 keyUsage         = digitalSignature,keyEncipherment
 extendedKeyUsage = clientAuth
+crlDistributionPoints = URI:@CRL@
```

Comment:
I would prefer that `crlDistributionPoints` is only added if `CRL` / `--crl` is set, otherwise this line isn't added.

Reply:
Is it possible to add conditions to the openssl cnf?

Reply:
No, you'll have to substitute the entire line as a string rather than just `@CRL@`.

```
@crlDistributionPoints = URI:CRL@
```